about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Shelby Willis - About</title>
+  <link rel="stylesheet" href="styles/styles.css">
+  <link rel="stylesheet" href="styles/about.css">
+  <link href="https://fonts.googleapis.com/css2?family=Yanone+Kaffeesatz:wght@300&display=swap" rel="stylesheet">
+  <script defer src="scripts/header.js"></script>
+</head>
+<body class="bg">
+  <header id="header" class="navlinksmain"></header>
+  <img src="images/coder2.jpg" width="450" height= "575">
+  <section class="container">
+  <section class="box">
+      <header> <h3 class="paragraphtitle">Hi I'm Shelby, your friendly neighborhood coder!</h3></header>
+      <div>I'm originally from Atlanta Georgia, currently residing in 
```

Comment:
Let's make this a `p` tag rather than a `div`. `div` elements are for dividing up similar elements, not for display text.